includes/reports/overview/list-table/class-sensei-reports-overview-list-table-abstract.php

```diff
@@ -339,6 +340,26 @@ protected function get_course_filter_value(): int {
 		return isset( $_GET['course_filter'] ) ? (int) $_GET['course_filter'] : 0;
 	}
 
+	/**
+	 * Get the orderby value.
+	 *
+	 * @return string orderby value.
+	 */
+	protected function get_orderby_value(): string {
```

Comment:
Looks like `private` visibility is enough for these methods (`get_orderby_value`, `get_order_value`).

Reply:
[changed](https://github.com/Automattic/sensei/pull/4973/commits/250a6ce515e2ba3467c681d2db9c80fd9ffb3350)